sdk/rust/src/api.rs

```diff
@@ -11,11 +11,21 @@ pub trait KMS {
         padding_mode: Option<&str>,
         digest_mode: Option<&str>) -> Result<String>;
 
+    async fn get_publickey(&mut self, _keyid: &str) -> Result<String>;
+    async fn sign(&mut self, _keyid: &str, _digest: &str) -> Result<String>;
+    
+    async fn verify(&mut self, _keyid: &str, _digest: &str, _signature: &str) -> Result<bool>;
+
     async fn decrypt(&mut self, _keyid: &str, _data: &str, _aad: Option<&str>) -> Result<String>;
 
     async fn encrypt(&mut self, _keyid: &str, _data: &str, _aad: Option<&str>) -> Result<String>;
```

Comment:
Not related to this PR: do we only support to set `aad`? What about `iv` in an AEAD? Also, why `_data` is `&str`(must be printable) rather than `&[u8]`(any bytes)

Reply:
iv is not exposed to user.

for the data type, it expected as based64 encoded.

Reply:
thanks for the explanation! I think it might be better to add some notes about this (I was confused so other might be)

Reply:
sure, we will add more comments for this.